lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -262,3 +270,62 @@ def refine_lines_centroid(lines, line_width):
         current += line_length
 
     return KymoLineGroup(interpolated_lines)
+
+
+def refine_lines_gaussian(lines, window=5, ignore_frames_with_overlap=True, initial_sigma=None):
```

Comment:
I wonder whether `ignore_frames_with_overlap` should have a default. It might be better if users consciously chose this. Also, in the future, when we do support resolving those, you don't want to suddenly change the default on people, so then we're stuck with ignoring them by default.

Reply:
Oh that's a good one. `overlap_strategy` may indeed be better. There's still quite a few things that could happen (shrink each window, fit overlapping Gaussians, something else entirely).

Reply:
Agreed, I like `overlap_strategy`. Changed it now so this argument excepts a string to define the behavior, can be easily augmented in the future

Reply:
Great job on the docstring it became perfectly clear what will happen!